clients/java/src/main/java/io/camunda/zeebe/client/impl/search/SearchQuerySortBase.java

```diff
@@ -15,7 +15,7 @@
  */
 package io.camunda.zeebe.client.impl.search;
 
-import io.camunda.zeebe.client.protocol.rest.SearchQuerySortRequest;
+import io.camunda.client.protocol.rest.SearchQuerySortRequest;
 import java.util.ArrayList;
 import java.util.List;
 
```

Comment:
💭 How about we only add new functionality in 8.6 to the `Camunda...` classes and leave the `Zeebe...` classes on 8.5-level? 🤔 Might be a nice incentive to move to the new classes. This might be a bit difficult now, maybe, since we're quite into the 8.6 cycle already. Just a thought.

Reply:
I thought about that, but it might be a lot time consuming

Reply:
Yeah, might be. We'd need to "reset" the `Zeebe...` classes to 8.5-level and only add those changes that are fixes 🙈 
On the other hand, we spare ourselves implementing everything twice now for 8.6. We have a lot of upcoming things to be added to the client for search and all the new endpoints we want to cover 😄

Reply:
We can simply copy the java client from 8.5, but we need to be sure that we didn't add any bugfixes since 8.5 release.

Reply:
I absolutely agree, yes.

Reply:
I will check that